content/SCALE/GettingStarted/VMandAppConfigSCALE.md

```diff
@@ -0,0 +1,71 @@
+
+### Set Up Shares
+
+Once the pool and datasets have been created, set up shares to enable data access. Three types of data sharing methods are discussed in 
+[Setting Up Data Sharing]({{< relref "SetUpSharing.md" >}}). More specific coverage of each share should be investigated based on your individual use case. 
+
+A good introduction as to how TrueNAS SCALE handles SMB shares is located at [SMB Shares Screens]({{< relref "SMBSharesScreens.md" >}}) and [Setting Up SMB Home Shares]({{< relref "AddSMBHomeShare.md" >}}). For a better understanding of how to configure Active Directory, see [Active Directory Screen]({{< relref "scale/scaleuireference/credentials/directoryservices/ActiveDirectory.md" >}}). Required Kerberos information is outlined here, and also in this article: [Configuring Kerberos]({{< relref "ConfigKerberosSCALE.md" >}}). Configuring LDAP best practices can be found at: [Configuring LDAP]({{< relref "ConfigLDAPSCALE.md" >}}).
+
+NFS basic share creation information can be found here: [Adding NFS Shares]({{< relref "AddingNFSShares.md" >}}). Adjust access permissions using the advanced options. 
+
+## Virtualization Prerequisites
+
+In order to be able to run a Virtual Machine (VM), hardware requirements include an x86 machine running a recent Linux kernel using an Intel processor with VT extensions, or an AMD processor with SVM extensions (AMD-V). To install a VM on SCALE, you need to first research the minimum and recommended specifications for the OS you plan to use and your full use case for that VM. Allocating too many resources to a VM can cause performance on the TrueNAS SCALE system to suffer, so it is recommended to plan ahead and ensure that the SCALE system has the resources to run both itself and a full VM deployment effectively.
+
+Software requirements include an installer for the OS you intend to install on the VM, and a TrueNAS storage pool. It is recommended to have additional datasets created beneath the storage pool to further organize your VM data.
+
+Review the [Virtualization Screens]({{< relref "VirtualizationScreens.md" >}}) to determine requirements for VM installation. See [Adding Managing VMs]({{< relref "CreatingManagingVMsSCALE.md" >}}) for more information on adding or managing VMs.
+
+## App Installation Prerequisites
+
+The first time you open the **Applications** screen, the UI asks you to choose a storage pool for applications. Select a pool that has enough space for all the application containers you intend to use. Set up a new dataset before installing your applications if you want to store your application data in a location separated from other storage on your system.
```

Comment:
Let the user know they don't have to set the pool the first time they access the Applications screen. They might not know they can cancel out of this process.
Also, you might want to let them know they can create their datasets before they go to this screen if that is what they want to do.
```suggestion
The first time you open the **Applications** screen, the UI asks you to choose a storage pool for applications. Either select a pool with enough space for all the applications you intend to use or you can cancel out of this and select the pool when you have all your storage set up the way you want it. Set up a new dataset before installing your applications if you want to store your application data in a location separated from other storage on your system.
```

Reply:
Revised to add additional information about pool selection.